<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FigureDesigner</class>
 <widget class="QDialog" name="FigureDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualization designer â€“ Coquery</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Design</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_13" stretch="0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1">
         <item>
          <widget class="QGroupBox" name="group_variables">
           <property name="title">
            <string>Available variables</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTableWidget" name="table_categorical">
              <column>
               <property name="text">
                <string>Categorical</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="table_numerical">
              <column>
               <property name="text">
                <string>Numerical</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="group_layout">
           <property name="title">
            <string>Layout</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QLineEdit" name="edit_figure_title">
              <property name="inputMask">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Figure title</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="1">
               <widget class="QWidget" name="widget_columns" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Column</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_2">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Sunken</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <item>
                     <widget class="QLineEdit" name="lineEdit_4">
                      <property name="placeholderText">
                       <string>(drop categorical here)</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QWidget" name="widget_rows" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Rows</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_3">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Sunken</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QLineEdit" name="lineEdit_5">
                      <property name="placeholderText">
                       <string>(drop categorical here)</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QWidget" name="widget_panel" native="true">
                <layout class="QGridLayout" name="gridLayout">
                 <item row="3" column="1">
                  <widget class="QLineEdit" name="edit_x_label">
                   <property name="placeholderText">
                    <string>X axis label</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="edit_y_label">
                   <property name="placeholderText">
                    <string>Y axis label</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QWidget" name="widget_data" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_8">
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>Data</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QFrame" name="frame_4">
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Sunken</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_9">
                       <item>
                        <widget class="QLineEdit" name="lineEdit_6">
                         <property name="placeholderText">
                          <string>(drop variable here)</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QFrame" name="frame_5">
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Sunken</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_10">
                       <item>
                        <widget class="QLineEdit" name="lineEdit_7">
                         <property name="placeholderText">
                          <string>(drop variable here)</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="group_figures">
           <property name="title">
            <string>Figure types</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QListWidget" name="list_figures">
              <property name="viewMode">
               <enum>QListView::IconMode</enum>
              </property>
              <property name="modelColumn">
               <number>0</number>
              </property>
              <property name="uniformItemSizes">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,2">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Legend</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="placeholderText">
               <string>Legend title</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Number of columns:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_columns">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>42</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Figure font</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QFontComboBox" name="combo_font_figure">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Color palette</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QRadioButton" name="radio_qualitative">
                <property name="text">
                 <string>Qualitative</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="combo_qualitative">
                <item>
                 <property name="text">
                  <string>Set1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Set2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Set3</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Paired</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Accent</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Pastel1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Pastel2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dark2</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="radio_sequential">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Sequential</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="combo_sequential">
                <item>
                 <property name="text">
                  <string>Greys</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Reds</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Greens</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Blues</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Oranges</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Purples</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>BuGn</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>BuPu</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>GnBu</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>OrRd</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>PuBu</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>PuRd</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>RdPu</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>YlGn</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>PuBuGn</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>YlGnBu</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>YlOrBr</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>YlOrRd</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QRadioButton" name="radio_diverging">
                <property name="text">
                 <string>Diverging</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="combo_diverging">
                <item>
                 <property name="text">
                  <string>RdBu</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>RdGy</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>PRGn</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>PiYG</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>BrBG</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>RdYlBu</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>RdYlGn</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Spectral</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="color_test_area">
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::InternalMove</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::MoveAction</enum>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectItems</enum>
              </property>
              <property name="movement">
               <enum>QListView::Snap</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>View</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_18">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QHBoxLayout" name="navigation_layout">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_placeholder">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_16" stretch="0,0">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_17">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Visualizing... </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progress_bar">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="box_visualize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
